api/peer-connection.ts

```diff
@@ -46,7 +46,7 @@ export class PeerConnection {
           ],
         },
       })),
-      audioTransceiver = pc.addTransceiver(this.returnAudioTrack, {
+      audioTransceiver = pc.addTransceiver('audio', {
```

Comment:
hmm was this change needed?

Reply:
hm no I had this for testing my weird connection drops and forgot to take it out